src/pages/docs/tutorials/tutorial-kitten-browser/app-setup/index.md

```diff
@@ -43,16 +43,19 @@ Let's give our module a name and establish its dependencies.  Edit `package.json
 {
 	"name": "enact-tutorial-kitten-browser",
 	"version": "1.0.0",
-	"description": "A general template for an Enact Moonstone application.",
+	"description": "A general template for an Enact Sandstone application.",
 	"author": "",
 	"main": "src/index.js",
 	"scripts": { [omitted] },
 	"license": "UNLICENSED",
 	"private": true,	
 	"repository": "",
 	"enact": {
-		"theme": "moonstone"
-	},
+        "isomorphic": true,
+		"ri": {
+			"baseSize": 48
+		}
+    },
```

Comment:
```suggestion
    	"theme": "sandstone"
    },
```

Reply:
Please check the indentation.
This should be fixed in your source code later too.

Reply:
OK. fixed in latest commit